app/views.py

```diff
@@ -51,8 +64,23 @@ def page_server_error(error):
 
 
 def convert_to_readable_time(time):
-    return time.strftime('%a %b %d,%Y %I:%M:%S %p')
-
-
-def display(forecast):
-    return 'Time: {}\nTemperature: {}°C\nHumidity: {}\nProbability of Precipitation: {}\nWeather Summary: {}'.format(convert_to_readable_time(forecast.time), round(forecast.temperature, 2), forecast.humidity, forecast.precipProbability, forecast.summary)
+    return time.strftime('%b/%d/%Y %a %I:%M:%S %p')
+
+
+def display(forecast, time_method):
+    if time_method == 'currently':
```

Comment:
again, you create a PR for *two* things, `currently` and other. small PRs please …